From 15b2e2d1b6142f9b6b9fd3b56c12a66241d62cb8 Mon Sep 17 00:00:00 2001
From: Chanh Nguyen <chanh@os.amperecomputing.com>
Date: Tue, 10 May 2022 06:56:43 +0000
Subject: [PATCH 1/2] sdbusplus: Remove the Error log in SMBus command sending

Some systems don't design the present pin for the NVMe slot,
the SendSmbusRWCmdRAW() will send out the LOG continuously in case
the NVMe not present. Should remove this LOG to increase trash log
in journalctl.

Signed-off-by: Chanh Nguyen <chanh@os.amperecomputing.com>
---
 smbus.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/smbus.cpp b/smbus.cpp
index a015714..852f6cc 100644
--- a/smbus.cpp
+++ b/smbus.cpp
@@ -107,11 +107,6 @@ int phosphor::smbus::Smbus::SendSmbusRWCmdRAW(int smbus_num, int8_t device_addr,
                                (unsigned char*)tx_data, rx_len,
                                (unsigned char*)Rx_buf);
 
-    if (res < 0)
-    {
-        fprintf(stderr, "Error: SendSmbusRWCmdRAW failed\n");
-    }
-
     res_len = Rx_buf[0] + 1;
 
     memcpy(rsp_data, Rx_buf, res_len);
-- 
2.25.1

